projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-top-posts.php

```diff
@@ -107,9 +93,7 @@ public function get_post_types() {
 	 */
 	public function get_top_posts( $request ) {
 		$period = $request->get_param( 'period' );
-		$number = $request->get_param( 'number' );
-		$types  = $request->get_param( 'types' );
-		return Jetpack_Top_Posts_Helper::get_top_posts( $period, $number, $types );
+		return Jetpack_Top_Posts_Helper::get_top_posts( $period, '', '' );
```

Comment:
maybe we could set default values for the two params so that we don't have to pass them here. Also considering the types of the params, could we please use `null` as default values? Thoughts?

Reply:
Makes total sense! Added now. :)